### DISCUSSION TITLE
what is the right printer for the photobooth

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#256] [Q&A] [answered]
I'm thinking about to add a photo printer to my booth.
any experinace ?

selphy cp 1500 ?

**Comment by mgineer85** [comment, marked as answer]
They all work in general. It's slow but can do the job. The older selphys (cp1300) seem to work better out of the box, but I don't have any experience with cp1500. Using a cp1300 myself.Am 02.05.2024 17:02 schrieb Striker62 ***@***.***>:
I'm thinking about to add a photo printer to my booth.
any experinace ?
selphy cp 1500 ?
